frontend/tests/playwright/product-list/filters.spec.ts

```diff
@@ -30,6 +30,19 @@ async function checkRefinementInSearchResult(facetName: string | null, facetOpti
    });
 }
 
+async function removeAndCheckRefinement(facetOptionValue: string | null | undefined, buttonText: string, page: Page) {
```

Comment:
Since we asserted that `facetOptionValue` exists before this, then the type should be a string only.

Reply:
The same can be applied to `checkRefinementInSearchResult`

Reply:
Oh, thanks for pointing it out! Updated it https://github.com/iFixit/react-commerce/pull/1244/commits/3cef31629fe015a433426eb01c3c04a28468a5d0.